imports/plugins/core/dashboard/client/components/ShopBrandMediaManager.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Components, composeWithTracker, registerComponent } from "@reactioncommerce/reaction-components";
+import { Media } from "/imports/plugins/core/files/client";
+
+/**
+ * ShopBrandMediaManager
+ * @param {Object} props Component props
+ * @returns {Node} React component
+ */
+function ShopBrandMediaManager(props) {
+  const { afterSetBrandImage, brandMedia, shop } = props;
+
+  if (!shop) return null;
+
+  const selectedMediaId = shop.brandAssets && shop.brandAssets.navbarBrandImageId;
+  const metadata = { type: "brandAsset", shopId: shop.internalId };
+
+  return (
+    <div>
+      <div className="rui gallery-thumbnails">
+        {(brandMedia || []).map((media) => (
+          <Components.ShopBrandImageOption
+            afterSetBrandImage={afterSetBrandImage}
+            key={media._id}
+            isSelected={selectedMediaId === media._id}
+            media={media}
+          />
+        ))}
+      </div>
+      <Components.MediaUploader canUploadMultiple metadata={metadata} />
+    </div>
+  );
+}
+
+ShopBrandMediaManager.propTypes = {
+  afterSetBrandImage: PropTypes.func,
+  brandMedia: PropTypes.arrayOf(PropTypes.shape({
+    _id: PropTypes.string.isRequired
+  })),
+  shop: PropTypes.shape({
+    _id: PropTypes.string,
+    brandAssets: PropTypes.shape({
+      navbarBrandImageId: PropTypes.string
+    })
+  })
+};
+
+const composer = (props, onData) => {
```

Comment:
Do we still want to use a `composer` here, or can we do this with an Apollo query hook?